samwise.js

```diff
@@ -1,26 +1,25 @@
 /**
```

Comment:
This line seems to be longer than our standard line length. Could you split it up for better readability?